### PR TITLE
Briefing Huboxing 2.0

### PR DESCRIPTION
**Pull request by DominikPanic**
:cl: LaKiller8
tweak: Briefing chairs can no longer be rotated at lightning speeds, all briefing objects are much tougher to destroy now.
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/windowdoor.dm [hunk 1]
```diff
@@ -315,3 +324,36 @@
 	dir = SOUTH
 	icon_state = "rightsecure"
 	base_state = "rightsecure"
+
+
+/obj/machinery/door/window/briefing/northleft
+	dir = NORTH
```
Comment (spookydonut) [inline code/game/machinery/doors/windowdoor.dm]: no.

File: code/game/machinery/doors/windowdoor.dm [hunk 2]
```diff
@@ -315,3 +324,36 @@
...
+	icon_state = "rightsecure"
+	base_state = "rightsecure"
+
+/obj/machinery/door/window/briefing/eastright
+	dir = EAST
+	icon_state = "rightsecure"
+	base_state = "rightsecure"
+
+/obj/machinery/door/window/briefing/westright
+	dir = WEST
```
Comment (spookydonut) [inline code/game/machinery/doors/windowdoor.dm]: very no

**Comment by spookydonut** [comment]
this is a hamfisted and poorly thought out attempt to stamp out something that has been arbitrarily ruled LRP.  I am against this change.

File: code/game/machinery/doors/windowdoor.dm [hunk 3]
```diff
@@ -252,6 +252,15 @@
 	health = 300.0 //Stronger doors for prison (regular window door health is 200)
 
 
+/obj/machinery/door/window/briefing
+	name = "reinforced safety windoor"
+	icon = 'icons/obj/doors/windoor.dmi'
+	icon_state = "leftsecure"
+	base_state = "leftsecure"
+	req_access = list(ACCESS_MARINE_BRIDGE)
+	health = 3000
```
Comment (Ghommie) [inline code/game/machinery/doors/windowdoor.dm]: this seems excessive, stick it to 1k or so if possible. Also give this a different description to hint the sturdiness.

File: code/game/objects/structures/window.dm [hunk 4]
```diff
@@ -411,6 +411,16 @@
 	health = 300
 	reinf = TRUE
 
+
+/obj/structure/window/reinforced/briefing
+	name = "reinforced safety glass"
+	desc = "A very tough looking glass window with a special rod matrice, probably bullet proof."
+	icon_state = "rwindow"
+	basestate = "rwindow"
+	health = 3000
```
Comment (Ghommie) [inline code/game/objects/structures/window.dm]: given how directional should have half the health of a full tile one and requisitions windows have 2k, I think 1k should be enough.

File: code/game/machinery/doors/windowdoor.dm [hunk 5]
```diff
@@ -252,6 +252,19 @@
...
+	name = "reinforced safety windoor"
+	desc = "An extremely strong windoor, capable of stopping the fiercest of riots."
+	icon_state = "leftsecure"
+	base_state = "leftsecure"
+	req_access = list(ACCESS_MARINE_BRIDGE)
+	health = 1000
+
+
+/obj/machinery/door/window/briefing/get_projectile_hit_chance(obj/item/projectile/P)
+	return TRUE
```
Comment (Ghommie) [inline code/game/machinery/doors/windowdoor.dm]: this should serve no purpose as windoors don't account for energy projectiles (yet), if I recall correctly.

File: code/game/objects/structures/window.dm [hunk 6]
```diff
@@ -411,6 +411,20 @@
...
+	name = "reinforced safety glass"
+	desc = "An extremely tough looking glass window with a special titanium rod matrice, most definitely bullet proof."
+	icon_state = "rwindow"
+	basestate = "rwindow"
+	health = 1500
+	reinf = TRUE
+
+
+/obj/structure/window/reinforced/briefing/get_projectile_hit_chance(obj/item/projectile/P)
+	return TRUE
```
Comment (Ghommie) [inline code/game/objects/structures/window.dm]:
```suggestion
	if((flags_atom & ON_BORDER) && P.dir != dir && P.dir != reverse_direction(dir))
		return FALSE
	else
		return TRUE

```
So bullets parallel to it shouldn't get stopped.

File: code/game/machinery/doors/windowdoor.dm (hunk 2 above)
Comment (Rohesie) [reply code/game/machinery/doors/windowdoor.dm]: This can be removed, it's not a must, if it makes the mappers' lives harder.

**Comment by Rohesie** [comment]
People will survive with being unable to rotate a chair a hundred times per second.
It won't detract from gameplay, it won't make rotating a chair for usual purposes any noticeably harder, nor it will impair creativity, which is what the game is about. It will merely limit one localized visual effect contrasting with the rest of the game.
Slippery slope is a fallacy. Take a stand against something that actually makes a difference.

**Comment by factoryman942** [comment]
local headmin can't spell "hugboxing"

**Comment by Alterist** [comment]
make a note somewhere that says your snowflake windows can't be shot through with lasers

**Pull request closed without merging**